### ISSUE TITLE
Error: MCP Version not found.. did forge change their format?

### ISSUE THREAD

**Issue by Jaemnie** [dscalzi/Nebula#69]
![image](https://github.com/dscalzi/Nebula/assets/121717917/2589e26d-f2db-431e-a758-b78ff04f5838)

In version 1.20.4 only, an error occurs while Nebula uses the npm start -- generate distro command.

**Comment by ManucrackYT** [comment]
i got the same issue

**Comment by dscalzi** [comment]
nope, PRs are welcome. My time to go on a forge code hunt is limited.

**Comment by ManucrackYT** [comment]
Alright

**Comment by BabyBear7** [comment]
Is there any update on the issue?

**Comment by ManucrackYT** [comment]
dscalzi won't fix it

**Comment by GeekCornerGH** [comment]
I'd like to mention that dscalzi never said he won't, but reverse engineering new forge format takes a while. If you manage to boot Minecraft 1.20.4+ on Helios, feel free to open PRs

**Comment by ManucrackYT** [comment]
The program hasn't been updated and it needs an important update. Dscalzi hasn't fixed it yet

**Comment by dscalzi** [comment]
Forge causes major headaches by needlessly changing everything, all the time. Open to PRs. My time to adapt to their changes is limited.

**Comment by dscalzi** [comment]
Related https://github.com/PrismLauncher/PrismLauncher/issues/1934

**Comment by dscalzi** [comment]
Fixed, BUT forge removed `-fml.modLists` which is how we load mods, so mods will not load for forge 1.20.4.

**For 1.20.4+, please switch to fabric or await NeoForged support**
